lmdeploy/model.py

```diff
@@ -220,7 +219,7 @@ def messages2prompt(self, messages, sequence_start=True):
         if isinstance(messages, str):
             return self.get_prompt(messages, sequence_start)
         system, users, assistants = self._translate_messages(messages)
-        ret = '<BOS>'
+        ret = '<BOS>' + self.system if system is None else '<BOS>' + system
```

Comment:
I didn't get it. What's the difference between `self.system` and `system`?
If `system is not None`, ret = '<BOS>' + '<BOS>' + system, that's weird.

Reply:
`system` is deduced from chat history passed in by user. When the user does not set it, it is None by default. While `self.system` is the value initialized in `model.__init__`.

in #464, when the user passed `"messages": [{"role": "system", "content": "将给定的英文(English)文本翻译成中文(simplified chinese)："}, {"role": "user", "content": "About the paper"}],` `system` will be used because it is not None.

Reply:
https://github.com/InternLM/lmdeploy/blob/e980377a206fa104cf1495d6d73af3ca2303f626/lmdeploy/model.py#L61

Reply:
Got you. Will fix it ASAP.